forms-web-app/src/lib/appeals-api-wrapper.js

```diff
@@ -72,9 +72,92 @@ exports.createOrUpdateAppeal = (appeal) => {
 };
 
 exports.getExistingAppeal = async (sessionId) => {
-  return handler(`/appeals/${sessionId}`);
+  return handler(`/api/v1/appeals/${sessionId}`);
 };
 
 exports.getLPAList = async () => {
   return handler('/api/v1/local-planning-authorities');
 };
+
+exports.EMPTY_APPEAL = {
```

Comment:
I would extract this also, `lib/empty-appeal.js` or something

Reply:
I'll do this thanks